scripts/buildkite/cache-s3.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, zlib, gmp }:
```

Comment:
why are you adding this file?

Reply:
It's called from default.nix.

Removed the file and it's reference form `default.nix` in dba088f